### DISCUSSION TITLE
VercelPostgresError: VercelPostgresError - 'missing_connection_string'

### DISCUSSION THREAD

**Discussion by gederer** [vercel/community#2385] [Help]
### Summary

I have two Vercel projects. Call them Project A and Project B. I also have a single Vercel Postgres database to which both projects are linked.

In the Dashboard, under the Settings tab, under the Environment Variables tab, I can see that all of my environment variables are set for both Project A and Project B. These variables include the ones I use to connect to Vercel Postgres (including `POSTGRES_URL`). I can connect to Vercel Postgres when running my app locally using the same environment variables and values. But, when I deploy to Vercel, I get the following in my logs:

```bash
 VercelPostgresError: VercelPostgresError - 'missing_connection_string': You did not supply a 'connectionString' and no 'POSTGRES_URL' env var was found.
    at (@qwik-city-plan.js:44:49224)
    at (@qwik-city-plan.js:44:49829)
    at (@qwik-city-plan.js:50:11887)
    at (@qwik-city-plan.js:50:7877)
    at (@qwik-city-plan.js:44:58480) {
  code: 'missing_connection_string',
  name: 'VercelPostgresError'
}
```

Any ideas?

**Comment by Schniz** [comment]
Hey @gederer as we DMed on Discord: I believe this is related to the Vite config in Qwik, as we can see that the environment variables are available on the edge function. Maybe @manucorporat will know the issue?
It seems that `process.env` is converted into a plain object. Maybe an adapter misconfiguration?
